app/react/kubernetes/cluster/NodeView/NodeApplicationsDatatable/NodeApplicationsDatatable.tsx

```diff
@@ -33,11 +32,19 @@ export function NodeApplicationsDatatable({
   );
   useRepeater(tableState.autoRefreshRate, onRefresh);
 
+  const envId = useEnvironmentId();
+  const {
+    params: { nodeName: nodeName },
+  } = useCurrentStateAndParams();
+
+  const applicationsQuery = useAllNodeApplicationsQuery(envId, nodeName);
+  const applications = Object.values(applicationsQuery.data ?? []);
+
```

Comment:
```suggestion
  const applications = applicationsQuery.data ?? [];
```
If it's an array already, Object.values shouldn't be required